pitest-entry/src/main/java/org/pitest/coverage/execute/DefaultCoverageGenerator.java

```diff
@@ -179,7 +179,9 @@ private void gatherCoverageData(final Collection<ClassInfo> tests,
 
       @Override
       public void apply(final CoverageResult cr) {
-        coverage.calculateClassCoverage(cr);
+        if (cr.isGreenTest() || !coverageOptions.getPitConfig().skipFailingTests()) {
```

Comment:
should we at least log that we are skipping coverage computation due to flag+failed tests?

Reply:
I actually don't know the strategy around logs in this project, just hacked for my immediate need to skip failing tests

Reply:
Where in the code can I see the recommended pattern to add logging?